gear-programs/checkpoint-light-client/src/wasm/utils.rs

```diff
@@ -82,3 +63,13 @@ pub fn construct_state_reply(
         replay_back,
     }
 }
+
+fn collect<'a, T: 'a + Copy>(
```

Comment:
I think it's better to move this inside of `construct_state_reply`

Reply:
ok